src/borrow_tracker/tree_borrows/tree/tests.rs

```diff
@@ -516,11 +516,11 @@ mod spurious_read {
         let source = LocStateProtPair {
             xy_rel: RelPosXY::MutuallyForeign,
             x: LocStateProt {
-                state: LocationState::new(Permission::new_frozen()).with_access(),
+                state: LocationState::new(false, Permission::new_frozen()).with_access(),
```

Comment:
Bare booleans are hard to read... maybe a `new_uninit` constructor would be useful?

Reply:
There are now two constructors, one for `init` and one for `uninit`